lco_ingester/fits.py

```diff
@@ -74,6 +74,11 @@ def check_catalog(self):
             # set obstype to CATALOG even though it's set to EXPOSE by the pipeline
             self.fits_dict['OBSTYPE'] = 'CATALOG'
 
+    def check_dayobs(self):
+        if not self.fits_dict.get('DAY_OBS'):
```

Comment:
I think the header is `DAY-OBS`, not `DAY_OBS`. Can you double check?

Reply:
Ah, yeah. I was confusing the archive's representation with the FITS header convention. Good catch.